Packs/MicrosoftExchangeOnPremise/Integrations/EWSv2/EWSv2.py

```diff
@@ -415,9 +417,7 @@ def prepare_context(credentials):  # pragma: no cover
 
 def prepare():  # pragma: no cover
     if NON_SECURE:
-        BaseProtocol.HTTP_ADAPTER_CLS = exchangelibSSLAdapter
-    else:
-        BaseProtocol.HTTP_ADAPTER_CLS = requests.adapters.HTTPAdapter
+        BaseProtocol.HTTP_ADAPTER_CLS = NoVerifyHTTPAdapter
```

Comment:
explain this change?

Reply:
fixing the error:
Cannot set verify_mode to CERT_NONE when check_hostname is enabled. (85)